docs/topics/js/js-debugging.md

```diff
@@ -0,0 +1,99 @@
+[//]: # (title: Debug Kotlin/JS code)
+
+JavaScript [source maps](https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/) provide mappings between
+the minified code produced by bundlers or minifiers and the actual source code a developer works with. This way, source
+maps enable support for debugging the code during its execution.
+
+Kotlin/JS Gradle plugin automatically generates source maps for the project builds, so they are available automatically
+without any additional configuration.
+
+## Debug in browser
+
+Most modern browsers provide tools that allow inspecting the page content and debugging the code that executes
+on it. Refer to your browser's documentation for more details.
+
+To debug Kotlin/JS in the browser:
+
+1. Run the project by calling one of the available `*Run` Gradle tasks, for example, `browserDevelopmentRun` or
+   `jsBrowserDevelopmentRun` in a multiplatform project.
+   Learn more in [this tutorial](running-kotlin-js.md#run-the-browser-target).
+2. Navigate to the page in the browser and launch its developer tools, for example, by right-clicking and
+   selecting the **Inspect** action. Learn how to [find the developer tools](https://balsamiq.com/support/faqs/browserconsole/)
```

Comment:
Not quite sure that this link is the best place we could lead to, but it's ok)

Reply:
Seb offered this as a single source of info about multiple browsers